js/MVP/containers/DocumentContainer.js

```diff
@@ -13,6 +13,7 @@ const DocumentWrapperStyled = styled.div`
   padding: 32px;
   background-color: white;
   margin-top: 32px;
+  margin-bottom: 32px;
```

Comment:
Jeg er redd denne ødelegger for andre komponenter som bruker denne komponenten.

- [ ] Fjern

Reply:
Jeg mener at jeg sjekket at det så bra ut på avlysning og referat. Hva tenker du at den skal ødelegge? 
Og har vi noen fast måte å gjøre dette på? Altså vertical spacing mellom komponenter?

Reply:
Jeg tror den kommer til å dytte ned i referat? Eller kollapser marginene? Jeg tenker det kommer en spacing på +32px nå.

Det jeg har tenkt er at hver komponent passer på sin egen top margin. På den måten er det ikke påtvunget margin mellom to komponenter, feks ved å legge til bottom margin her vil man ikke enkelt kunne ha spacing på 16px mellom denne komponenten og en annen hvis man ønsker det.

Reply:
Dersom det er behov for justerbar styling/margins på denne komponenten, så kan man jo alltids sende dette inn som en prop. Da får man faste verdier på stylingen. (marginTop?: boolean)

Reply:
Det kan funke det hvis det er behov flere steder.

Reply:
Jeg fjerna margin-bottom og la til margin-top et annet sted. Ble bedre da.